draft-ietf-quic-recovery.md

```diff
@@ -1139,8 +1145,9 @@ OnAckReceived(ack, pn_space):
     OnPacketAcked(acked_packet.packet_number, pn_space)
 
   DetectLostPackets(pn_space)
-
-  pto_count = 0
+  // Reset pto_count, unless the client receives an Initial ACK
+  if (endpoint is server || pn_space != Initial)
```

Comment:
Shouldn't we better invoke `PeerCompletedAddressValidation()`?

Reply:
Done.